torch/_logging/_internal.py

```diff
@@ -250,7 +255,26 @@ def set_logs(
     modules = modules or {}
 
     def _set_logs(**kwargs):
+        if "all" in kwargs and kwargs["all"]:
+            default_level = kwargs["all"]
```

Comment:
nit
```suggestion
        default_level = kwargs.pop("all", None)
        if default_level:
```

and remove the `elif alias == "all"` below.